sql/core/src/main/scala/org/apache/spark/sql/execution/columnar/ColumnStats.scala

```diff
@@ -307,7 +308,7 @@ private[columnar] final class DecimalColumnStats(precision: Int, scale: Int) ext
       // TODO: this is not right for DecimalType with precision > 18
       gatherValueStats(value)
     } else {
-      gatherNullStats
+      gatherNullStats()
```

Comment:
Yeah this was mostly from the scala style guide since this mutates the backing stats. http://docs.scala-lang.org/style/method-invocation.html#arity-0
I don't have a strong opinion though, so happy to swap it back.

Reply:
Let's just swap it back to make the diff small.